_rules/SC-1-3-1-required-aria-state-or-property-has-valid-value.md

```diff
@@ -21,26 +21,27 @@ authors:
 
 ### Applicability
 
-Any [non-empty](#non-empty) [WAI-ARIA 1.1 state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is specified on an HTML or SVG element.
+Any [WAI-ARIA required state or property](https://www.w3.org/TR/wai-aria-1.1/#requiredState) that 
```

Comment:
The applicability now only refers to aria-controls and no other property. We could just state that in the applicability and the name of the rule.

Reply:
@ShadowBB , we discussed internally if that was the way forward for this rule. 
While it's the only required property in WAI-ARIA 1.1, there could potentially also be roles and requirements specified in other standards, e.g. EPUB. 
Having "broader" scoped rule could actually catch something in other standards, that maps to a completely different success criterion, I guess.
So I guess I am for your suggestions of limiting the rule to explicitly `aria-controls`, and then we can be completely sure what we are testing...

Reply:
aria-controls is required for the roles `scrollbar` and `combobox`. I have now limited the rule to only look for `aria-required` on role `scrollbar`, since `combobox` is a story of its own, see https://github.com/w3c/aria/issues/716.